### ISSUE TITLE
I want clickable cards to be more accessible

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#3396]
## Problem/Motivation
When viewing cards on this page, the "Learn more" links are not descriptive enough.
https://registrar.arizona.edu/faculty-staff-resources/room-course-scheduling/schedule-classes

### Describe the bug
Card link text field should be more helpful for the content owner.

### To Reproduce
1. Create a card with a link
2. Use screen reader to navigate to the clickable button
3. Observe link text that is redundant and not explicit

## Proposed resolution
1. Add help text and/or warning to the card link text field
4. Add link around the card title when using the stretched link

https://github.com/az-digital/az_quickstart/tree/issue/3396

### Expected behavior
When using a screen reader, link should have useful context.

## Additional context
Issue seen on both of these pages
- Initial page: https://registrar.arizona.edu/faculty-staff-resources/room-course-scheduling/schedule-classes
- Mockup with standard Quickstart cards: https://live-dana-qs2.pantheonsite.io/schedule-classes-title

**Comment by danahertzberg** [comment]
Updated the template, but unsure where to find just the URL to put in the path. Only find {{ link }} which is the URL and link title.

**Comment by danahertzberg** [comment]
Add warning message if all cards have "read more" or same text in the link text field.

Update link URI here: https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_card/src/Plugin/Field/FieldFormatter/AZCardDefaultFormatter.php#L241-L247

Add link around heading and add help text to form. Warning not needed right now.
